master/static/migrations/20220512160917_update-job-type-jobs.tx.up.sql

```diff
@@ -0,0 +1,17 @@
+ALTER TABLE public.jobs
```

Comment:
take a look at how `static/migrations/20210716133048_add-deleting-states.tx.up.sql` is done. this should work fine, too though, but that saves one `alter table`.